src/applications/medical-copays/sass/medical-copays.scss

```diff
@@ -1 +1,5 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.sub-title {
+  font-size: 20px;
```

Comment:
We can just use the CSS utility here instead

`vads-u-font-size--lg`

Reply:
oh ya ill update it tks!